### ISSUE TITLE
Enhance `istioctl verify-install` to consume operator icp cr or profile

### ISSUE THREAD

**Issue by irisdingbj** [istio/istio#18547]
Thanks @linsun for this idea.

Currently `istioctl verify-install` has two functions
`istioctl very-install` - perform install pre-check against cluster
`istioctl verify-install -f, --filename strings   Istio YAML installation file.` -perform post install verification against a cluster

Now we support install Istio via `istioctl manifest apply` it is better we add below two functions into `istioctl verify-install`

```
istioctl verify-install -f --filename  path to Istio YAML installation file or 
Path to file containing IstioControlPlane CustomResource
```
```
istioctl verify-install -p --profile  profile name for Istio installation
```

@ostromart @linsun @sdake @esnible @howardjohn 

Please review above and looking forward to your comments.

**Comment by esnible** [comment]
@irisdingbj Will you have this implemented for 1.5?

**Comment by esnible** [comment]
@irisdingbj Should I try to take this over?

**Comment by ostromart** [comment]
If I understand the status quo of this issue, the current plan is to write the IstioOperator CR to the cluster during manifest apply and use that for istioctl verify-install? I hope so because this is quite complementary to other features we may want to add that require this state from the cluster so +1 from me.

**Comment by esnible** [comment]
The `-f` support is implemented and will be in 1.6.0-alpha.2.
We have also added `--release <release>` support.

Because 1.6.0.alpha.1 records the deployed state to an in-cluster IstioOperator there is no need for the user to use `--profile`.  That is the only feature missing from this issue, and no longer needed, so I am closing the issue.
